### PR TITLE
OT collector for k8s

### PR DESCRIPTION
**Pull request by yyyogev**
# What changed

Added a config to deploy OT's collector on k8s along with instructions  set up a secret of the traces token

----

## Pages to review

- LinkToPage1
- LinkToPage2

## Remaining work

- [ ] Technical review
- [ ] Copy Review
- [ ] Redirects: 

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information:
- [ ] Replace original content on the support portal with 'this page has been moved to ...' - paste the URL
- [ ] Update these log shipping pages in the app: - paste the URL

### CODE REVIEW

File: _source/user-guide/distributed-tracing/k8s-deployment.md [hunk 1]
```diff
@@ -64,19 +236,24 @@ spec:
             port: 14269
         env:
         - name: ACCOUNT_TOKEN
-          value: <<ACCOUNT_TOKEN>> # Replace with the Tracing account token from Logz.io in Manage accounts > Distributed Tracing
```
Comment (YajB) [inline _source/user-guide/distributed-tracing/k8s-deployment.md:67]: @yyyogev - 
Should this line include the comment
`# Replace with the Tracing account token from Logz.io in Manage accounts > Distributed Tracing`

Comment (YajB) [inline _source/user-guide/distributed-tracing/k8s-deployment.md:67]: @yyyogev 
How does fetching the account token work now? 
Does the code get it automatically, without user intervention?

Comment (yyyogev) [reply _source/user-guide/distributed-tracing/k8s-deployment.md:67]: from step 1

**Comment by yyyogev** [comment]
@YajB I think we should split it into tabs, I just didn't know how to do it

**Review by YajB — APPROVED** [review]

**Comment by YajB** [comment]
> @YajB I think we should split it into tabs, I just didn't know how to do it

@yyyogev  -- 
Which sections should be in different tabs? 

 Can you point me to an example in our docs of what you mean?

**Comment by yyyogev** [comment]
tab 1 - OpenTelemetry collector + Jaeger agents
tab 2 - Jaeger collector and agents

**Pull request merged**